app/app.go

```diff
@@ -688,7 +688,7 @@ func newETH2Client(ctx context.Context, conf Config, life *lifecycle.Manager,
 	if conf.SimnetBMock { // Configure the beacon mock.
 		const dutyFactor = 100 // Duty factor spreads duties deterministically in an epoch.
 		opts := []beaconmock.Option{
-			beaconmock.WithSlotDuration(time.Second),
+			beaconmock.WithSlotDuration(time.Second * 6),
```

Comment:
why this?

Reply:
I tested with slot times 3s and 6s. And found out this large cluster is stable with 6s slot times. Whereas in 3s slot times, I observed 70% of duties were failing and those that were passing had broadcast delay of around 9s.
With 6s slot times the cluster had broadcast delays of around 5.5s to 6.5s. This is only for simnet/compose run and not for simnet_test.

Reply:
I would prefer 1s for compose by default, we can do 6s only for this test?

Reply:
Done!